scenes/music_puzzle/bard.gd

```diff
@@ -3,22 +3,10 @@
 class_name Bard
 extends Talker
 
-@export var xylophone: MusicalRockXylophone
+@export var puzzle: MusicPuzzle
 
+var first_conversation: bool = true
```

Comment:
@pablitar since you are working on game state (#9) – I wonder if this the kind of thing you would expect to live there, rather than as an ad-hoc boolean on this NPC?

Reply:
It kinda depends. If the state should outlive the node, then yes. If not, then no. If the quest is replayable, maybe we don't want this stuff to be in the global state.

Reply:
Eventually there may be a need for "scoped" state. For instance, state that survives during the lifecycle of a StoryQuest. But we should follow up on that once we have a clearer picture of what state do we need.

Reply:
The other option is to look at the game state from the player point of view. "Have I already talked to the bard?" Which suggests that the state should be global.

Reply:
Or a property of the player...!